### PR TITLE
Add chefkoch.de parser

### PR DESCRIPTION
**Pull request by tahesse**
Quickly added a parser for https://www.chefkoch.de/

URL used for tests: https://www.chefkoch.de/rezepte/1170311223132029/Hackbraten-supersaftig.html - yummy!

### CODE REVIEW

**Comment by tahesse** [comment]
@maintainers Kindly let me know if d6bc023 is inappropriate for this PR scope. TYSM! :smile:

File: README.rst [hunk 1]
```diff
@@ -184,6 +185,12 @@ Assuming you have ``python3`` installed, navigate to the directory where you wan
     python -m coverage run -m unittest &&
     python -m coverage report
 
+In case you want to run a single unittest for a newly developed scraper
+
+.. code::
+
+    python -m coverage run -m unittest tests.test_myscraper
```
Comment (bfcarpio) [inline README.rst:192]: Things like this are always appreciated! Thanks!

File: recipe_scrapers/chefkoch.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from ._abstract import AbstractScraper
+
+
+class Chefkoch(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"chefkoch.{domain}"
```
Comment (bfcarpio) [inline recipe_scrapers/chefkoch.py:7]: While I'm not opposed to using the `kwarg`, we do this when there are multiple domains to the same website. Take a look at [this example](https://github.com/hhursev/recipe-scrapers/blob/60ff6a52ce81eafae79b27c4775474ce3b979fea/recipe_scrapers/__init__.py#L103).

Is there another domain that this website is available at?

**Review by bfcarpio — CHANGES_REQUESTED** [review]
Thank you for this contribution. One small convention based comment, but let me know what you think.

File: recipe_scrapers/chefkoch.py (hunk 2 above)
Comment (tahesse) [reply recipe_scrapers/chefkoch.py:7]: There is none. It's just "chefkoch.de". Do you want me to remove the `domain` parameter?

Comment (bfcarpio) [reply recipe_scrapers/chefkoch.py]: Yes, I'd prefer that we maintain the style of the package and only use that pattern when there are multiple TLDs possible.

**Review by bfcarpio — APPROVED** [review]
This looks good! Thank you for the quick change.

**Pull request merged**